tests/tasks/generators/clusterloader/load-slos.yaml

```diff
@@ -162,6 +162,8 @@ spec:
       fi
       cat $(workspaces.source.path)/perf-tests/clusterloader2/testing/load/config.yaml
       cd $(workspaces.source.path)/perf-tests/clusterloader2/
+      kubectl --kubeconfig=$KUBECONFIG delete servicemonitor kube-dns -n monitoring
+      kubectl --kubeconfig=$KUBECONFIG delete servicemonitor core-dns -n monitoring
```

Comment:
This one should be coredns with no `-`: https://github.com/kubernetes/perf-tests/blob/e924883aeecbb2fb01c53813c6eff1210e161bbc/clusterloader2/pkg/prometheus/manifests/default/prometheus-serviceMonitorCoreDNS.yaml#L6